apps/rule-manager/cfn/cfn.yaml

```diff
@@ -0,0 +1,211 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Typerighter rule-management tool
+Parameters:
```

Comment:
Do we want to check our generated YAML in? Or is this something else?

Reply:
This was leftover from when I was implementing the cfn as a yaml file. It's no longer needed and I will remove it.